lisa/operating_system.py

```diff
@@ -215,53 +215,96 @@ def capture_system_information(self, saved_path: Path) -> None:
     @classmethod
     def _get_detect_string(cls, node: Any) -> Iterable[str]:
         typed_node: Node = node
-        cmd_result = typed_node.execute(cmd="lsb_release -d", no_error_log=True)
-        yield get_matched_str(cmd_result.stdout, cls.__lsb_release_pattern)
+        cmd_result = typed_node.execute(
+            cmd="lsb_release -d", no_error_log=True, timeout=30
+        )
+        yield cls._get_matched_str_from_cmd_result(
```

Comment:
Use it as a function, not in the class. Because the function is not related to the class. Try shorter name like `get_matched_os_name`?

Reply:
I wanted to add function in the same file lisa\util\__init__.py as get_matched_str. But it has a circle import due to adding ExecutableResult. Then I added get_matched_os_name in operating_system.py.

Reply:
You can add the method in this file, but not in this class.